ledger/sandbox/src/main/resources/db/migration/postgres/V9__Contract_Divulgence.sql

```diff
@@ -0,0 +1,17 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V9: Contract divulgence
+--
+-- This schema version complements V8 by altering the contracts_divulgences.contract_id foreign key
+-- to reference contract_data.id (instead of contracts.id)
+--
+-- This is done because for divulged contracts, we only know the contract data,
+-- but no other metadata in contracts.
+---------------------------------------------------------------------------------------------------
+
+-- due to lack of support for dropping or modifying referential integrity constraints in h2
+-- rebuild the contract_divulgences table:
+
+ALTER TABLE contract_divulgences RENAME TO contract_divulgences_to_be_dropped;
```

Comment:
Was this change purposefully put in this PR?

Reply:
Good catch - I must have done this accidentally switching between branches. Removing.